openshift_images/index.adoc

```diff
@@ -0,0 +1,127 @@
+[id="overview-of-images"]
+= Overview of images
+include::modules/common-attributes.adoc[]
+:context: overview-of-images
+
+toc::[]
+
+[discrete]
```

Comment:
Unless the overview assemblies are not supposed to have IDs, this should be a unique ID.